tinkerbell/boots/templates/rolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.boots.deploy }}
```

Comment:
Do we want to make clear the fact these are cluster scoped resources in the file name?

Reply:
sure, i need to add some docs and will be sure to note this.